include/picongpu/particles/creation/SpawnFromSourceSpecies.kernel

```diff
@@ -156,6 +156,13 @@ namespace picongpu::particles::creation
             // create shared memory variable for kernel state
             PMACC_SMEM(worker, sharedKernelState, typename Modules::KernelStateType);
 
+            // create cache
+            auto cache = Modules::template InitCacheFunctor<T_KernelConfigOptions...>::getCache(
+                worker,
+                superCellIndex,
+                std::forward<T_AdditionalData>(additonalData)...);
+            // worker.sync() not necessary since shared memory init may be executed in parallel
```

Comment:
I do not understand why sync is not required since init is executed in parallel.

Reply:
offline discussion, I wanted to note that I skip the thread sync after the collective cache init, because we sync after completing the shared memory init anyway and the shared memory init does not depend on the cache init.